client/verta/verta/_pipeline_orchestrator/_orchestrator.py

```diff
@@ -189,12 +252,15 @@ def _init_step_handlers(
             Mapping of step names to their handlers.
 
         """
+        step_inputs = cls._get_steps_input_names(pipeline_spec)
+
         step_handlers = dict()
         for step in pipeline_spec["steps"]:
-            step_name = step["attributes"]["name"]
-            step_handlers[step_name] = ModelObjectStepHandler(
-                name=step_name,
-                predecessors=pipeline_spec["graph"]["inputs"].get(step_name, []),
-                model=ModelObjectStepHandler._init_model(conn, step["rmvId"]),
+            step_handlers[step["name"]] = ModelObjectStepHandler(
+                name=step["name"],
+                predecessors=step_inputs.get(step["name"], []),
+                model=ModelObjectStepHandler._init_model(
+                    conn, step["model_version_id"]
+                ),
```

Comment:
Before, I was using the pipeline schema from our design doc. This is updated to use the schema Cory ended up defining in Swagger (when our design doc met the constraints of reality).